prototype/link/scripts/PlayVideo_analyses/supervisor_Mickey.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+#
+# Automate the analysis of PlayVideo 
+# Assumes you are running from the subject base directory
+
+# Step 1: Z-score data while ignoring NaNs (already done in earlier preprocessing)
+# Step 2: Align the participant data to adult standard 
+# Step 3: Extend the data if there are TRs missing at the end
+# Step 4: figure out which TRs they had their eyes closed for 
+# Transfers all of these files to the group folder
+#
+# Reworked so everything can be run in participant folder TY 052021
+#
+#SBATCH --output=./logs/supervisor_Mickey-%j.out
+#SBATCH -p day
+#SBATCH -t 1:00:00
+#SBATCH --mem 16000
+
+if [ $# -eq 0 ]
+then
+    analysis_type='default'
+else
+    analysis_type=$1
+fi
+
+preprocessing_type='linear_alignment' # could also be "nonlinear"
+
+source globals.sh
+
+# What is the name of movie you care about?
+movie='PlayVideo_'
+movie_out_name='Mickey' # Mickey is a more informative name 
+nTRs=148 # assume two back-to-back viewings
```

Comment:
Maybe add more detail here and in the title to explain the discrepancy between the names PlayVideo and Mickey. E.g., we only ever showed them Mickey using this stimulus presentation method